plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSystemTables.java

```diff
@@ -64,6 +64,19 @@ public void setUp()
         assertUpdate("INSERT INTO test_schema.test_table_nan VALUES (1, 1.1, 1.2, CAST('2022-01-01' AS DATE)), (2, nan(), 2.2, CAST('2022-01-02' AS DATE)), (3, 3.3, nan(), CAST('2022-01-03' AS DATE))", 3);
         assertUpdate("INSERT INTO test_schema.test_table_nan VALUES (4, nan(), 4.1, CAST('2022-01-04' AS DATE)), (5, 4.2, nan(), CAST('2022-01-04' AS DATE)), (6, nan(), nan(), CAST('2022-01-04' AS DATE))", 3);
         assertQuery("SELECT count(*) FROM test_schema.test_table_nan", "VALUES 6");
+
+        assertUpdate("CREATE TABLE test_schema.test_table_with_dml (_varchar VARCHAR, _date DATE) WITH (partitioning = ARRAY['_date'])");
+        assertUpdate(
+                "INSERT INTO test_schema.test_table_with_dml " +
+                        "VALUES " +
+                        "('a1', DATE '2022-01-01'), ('a2', DATE '2022-01-01'), " +
+                        "('b1', DATE '2022-02-02'), ('b2', DATE '2022-02-02'), " +
+                        "('c1', DATE '2022-03-03'), ('c2', DATE '2022-03-03')",
+                6);
+        assertUpdate("UPDATE test_schema.test_table_with_dml SET _varchar = 'a1.updated' WHERE _date = DATE '2022-01-01' AND _varchar = 'a1'", 1);
+        assertUpdate("DELETE FROM test_schema.test_table_with_dml WHERE _date = DATE '2022-02-02' AND _varchar = 'b2'", 1);
+        assertUpdate("INSERT INTO test_schema.test_table_with_dml VALUES ('c3', DATE '2022-03-03'), ('d1', DATE '2022-04-04')", 2);
+        assertQuery("SELECT count(*) FROM test_schema.test_table_with_dml", "VALUES 7");
```

Comment:
What's the relationship between new columns in `$manifests` and this test change?

Reply:
This change showcases in further detail how the data files row statistic fields are filled in a DML(UPDATE/DELETE) scenario.